postfix.yaml

```diff
@@ -1,31 +1,35 @@
 package:
   name: postfix
-  version: "3.10.4"
-  epoch: 0
-  description: Secure and fast drop-in replacement for Sendmail (MTA)
+  version: ${{vars.package_version}}
+  epoch: 1
+  description: Secure and fast drop-in replacement for Sendmail (MTA) with Rspamd integration
   copyright:
     - license: IPL-1.0 OR EPL-2.0
-  scriptlets:
-    pre-install: |
-      #!/bin/sh
-      addgroup -S postfix 2>/dev/null
-      addgroup -S postdrop 2>/dev/null
-      adduser -S -H -h /var/spool/postfix -G postfix -g postfix postfix 2>/dev/null
-      addgroup postfix mail 2>/dev/null
-      adduser -S -H -h /var/mail/domains -s /sbin/nologin -G postdrop -g vmail vmail 2>/dev/null
-
-      exit 0
   checks:
     disabled:
       - setuidgid
+      - usrmerge
   dependencies:
     runtime:
+      - bash
+      - busybox
+      - cronie
       - merged-usrsbin
       - rspamd
+      - rsyslog
       - sasl-xoauth2
       - supervisor
       - wolfi-baselayout
 
+# Dual-source package: scripts from github (docker-postfix), source code from official Postfix tarball
```

Comment:
I'm not confident on this tbh, it feels like these should be separate packages instead. Its a bit unusual approach and would make hard to maintain. But happy to discuss more about on this.